### DISCUSSION TITLE
New features

### DISCUSSION THREAD

**Discussion by GWPvGWP** [macedonga/beunblurred#55] [Ideas]
- Discovery seems to be back
- Users can take videos

Can these features be implemented? It's a bit early maybe, because not all users have these features already.

**Comment by macedonga** [comment]
I'll try to check those out once I fix the login :-)

**Comment by GWPvGWP** [comment]
As far as I know these features haven't made it and are not rolled out to all users in the original app.

**Reply by macedonga** [reply]
Yeah, unfortunately I still don't have access to them
